orchestra/migrations/0080_completed_project_status.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Testing (send me this on slack, not the public comment):
1) Before and after counts of project statuses
2) Before running migration, run the old and new versions of `completed_projects`. After running migration, run the old and new versions of `completed_projects`. There should be complete overlap between all 4 sets.
3) Before running migration, run the old and new versions of `incomplete_projects`. After running migration, run the old and new versions of `incomplete_projects`. There should be complete overlap between all 4 sets.